### PR TITLE
Create jenkins file

### PR DESCRIPTION
**Pull request by Cxristian**

### CODE REVIEW

File: gradle.properties [hunk 1]
```diff
@@ -1,8 +1,8 @@
 ## CONFIGURE THE APP CLIENT.
-client_id =
-client_secret =
-user_name =
-password =
-security_token =
+client_id = 3MVG9IHf89I1t8hr66Zq39.7RKKVh5shKSUTpAo12udn.agfJ_m0Wjb8w4xOsdO_7KCD.sASrFEGhD8tw0_9H
```
Comment (alszla) [inline gradle.properties]: Oooops!

File: Jenkinsfile [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+    post {
+        always {
+            echo 'This is a post action.'
+            mail to: 'Cristian.lujan@fundacion-jala.org',
+                 subject: "Test Pipeline: ${currentBuild.fullDisplayName}",
+                 body: "The pipeline ${currentBuild.fullDisplayName} completed successfully."
+        }
+    }
+}
```
Comment (TheYisusGithub) [inline Jenkinsfile]: Please put one salt of lien here.

File: gradle.properties [hunk 3]
```diff
@@ -4,5 +4,5 @@ client_secret =
 user_name =
 password =
 security_token =
-auth_url = https://login.salesforce.com/services/oauth2/token
-tester_name =
+auth_url =
+tester_name = Cxriss
```
Comment (MelissaRoman06) [inline gradle.properties]: Please delete remaining credentials

File: src/test/resources/features/opportunity/DeleteOpportunity.feature [hunk 4]
```diff
@@ -4,18 +4,18 @@
...
           | message | [entity is deleted] |
 
     @Negative
     Scenario: Delete a non-existent opportunity
       When User makes a DELETE request for opportunity 00T3i000005aHO0EAM
-      Then User get a "404" status code as response
-      And The message of the response is:
+      Then the status code is 404
+      And the response includes the following
         | message | [The requested resource does not exist] |
```
Comment (titor-oopart) [inline src/test/resources/features/opportunity/DeleteOpportunity.feature]: add empty line

**Review by titor-oopart — APPROVED** [review]

**Review by alszla — APPROVED** [review]

**Pull request merged**